src/main/kotlin/com/openlattice/organization/OrganizationExternalDatabaseApi.kt

```diff
@@ -0,0 +1,167 @@
+package com.openlattice.organization
+
+import retrofit2.http.*
+import java.util.*
+
+interface OrganizationExternalDatabaseApi {
```

Comment:
How about just DatasetApi?

Reply:
I use "OrganizationExternalDatabase" all over the place in this feature, was trying to keep it consistent. I'm happy to change it though, I realize it's a super verbose name.